demo/screens/ListSellOrdersScreen.js

```diff
@@ -49,32 +50,42 @@ export default class ListLandsScreen extends React.Component {
     const hasOrder = parseInt(orderId, 16) !== 0;
     if (!hasOrder) throw Error(`Estate (id:${estateId}) has no sell order.`);
 
-    const priceMana = Number(price.toString()) / 1e18;
+    // Note: Conversion to USD will be implemented on v0.2.0
     const manaPerUsd = 30;
+    const priceMana = Number(price.toString()) / 1e18;
     const priceUsd = Number(priceMana / manaPerUsd).toFixed(2);
     const imgUrl = `https://api.decentraland.org/v1/estates/${estateId}/map.png`;
 
     return {
-      id: estateId,
-      name: estateName,
+      id: orderId,
       priceMana,
       priceUsd,
-      img: imgUrl,
-      orderId,
       seller,
       expiresAt,
+      asset: {
```

Comment:
Is it asset so that it could work with a different ERC721 NFT as well? Or just land also? Or will asset here always be an estate in the future?

Reply:
Marketplace contract accept any ERC721 as asset, on Decentraland context, it can be parcel or estate, I'm not sure if it's the best approach, I've thought on both solutions:
Using `asset` field with some type field to indicate which asset it is, 
or
Using `estate` / `parcel` fields and check which isn't `undefined` to know which asset it is.
What do you think?

Reply:
I think I prefer asset field with an enumerable list of strings for type.

Reply:
I've created a comment about that.

Reply:
Thanks. Resolving.